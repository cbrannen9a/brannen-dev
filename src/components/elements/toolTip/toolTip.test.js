import React from 'react';
import renderer from 'react-test-renderer';

import ToolTip from './toolTip';

const message = 'Test Tooltip';

describe('ToolTip', () => {
    it('renders correctly', () => {
        const tree = renderer
            .create(
                <ToolTip message={message}>
                    <div>Child</div>
                </ToolTip>
            )
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});
