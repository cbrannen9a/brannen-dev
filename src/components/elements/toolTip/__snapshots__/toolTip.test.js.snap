// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ToolTip renders correctly 1`] = `
<span
  className="tooltip"
  onMouseLeave={[Function]}
>
  <span
    className="tooltip-trigger"
    onMouseOver={[Function]}
  >
    <div>
      Child
    </div>
  </span>
</span>
`;
